ai-ml/trainium-inferentia/eks.tf

```diff
@@ -498,7 +548,7 @@ module "eks" {
       # aws ssm get-parameters --names /aws/service/eks/optimized-ami/1.27/amazon-linux-2-gpu/recommended/image_id --region us-west-2
       # ami_id   = "ami-0e0deb7ae582f6fe9" # Use this to pass custom AMI ID and ignore ami_type
       ami_type       = "AL2_x86_64_GPU"
-      capacity_type  = "SPOT"
+      capacity_type  = var.inf2_capacity_type
```

Comment:
Keep it hard-coded this value. Don't create a variable for this

Reply:
Sure, will do.